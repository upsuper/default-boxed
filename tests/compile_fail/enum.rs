use default_boxed::DefaultBoxed;

#[derive(DefaultBoxed)]
enum Foo {}

fn main() {}
