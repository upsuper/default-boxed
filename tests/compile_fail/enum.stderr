error: only structs are supported
 --> $DIR/enum.rs:4:1
  |
4 | enum Foo {}
  | ^^^^
